src/StackExchange.Redis/Interfaces/IDatabase.cs

```diff
@@ -1419,6 +1419,53 @@ public interface IDatabase : IRedis, IDatabaseAsync
         /// <remarks>https://redis.io/commands/zlexcount</remarks>
         long SortedSetLengthByValue(RedisKey key, RedisValue min, RedisValue max, Exclude exclude = Exclude.None, CommandFlags flags = CommandFlags.None);
 
+        /// <summary>
+        /// Returns a random element from the sorted set value stored at <paramref name="key"/>.
+        /// </summary>
+        /// <param name="key">The key of the sorted set.</param>
+        /// <param name="flags">The flags to use for this operation.</param>
+        /// <returns>The randomly selected element, or <see cref="RedisValue.Null"/> when <paramref name="key"/> does not exist.</returns>
+        /// <remarks>https://redis.io/commands/zrandmember</remarks>
+        RedisValue SortedSetRandomMember(RedisKey key, CommandFlags flags = CommandFlags.None);
```

Comment:
Noodling on this, probably want to have a `SortedSetRandomMemberWithScore` method? Where you just pass in `COUNT` 1 and the `WITHSCORES` argument to yield a single SortedSetResult.

Reply:
I was thinking: it's rare enough I'm okay leaving it off - if there was ever a complaint, we could add it later.

Reply:
@slorello89 If good with the above, I'll go ahead and get this in - thoughts?

Reply:
👍